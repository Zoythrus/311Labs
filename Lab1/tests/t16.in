merge
5 7
11
26
39
102
123
6
17
26
54
73
73
81
