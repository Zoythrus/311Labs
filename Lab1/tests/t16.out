6
11
17
26
26
39
54
73
73
81
102
123
